sql/world.sql

```diff
@@ -156496,7 +156496,7 @@ INSERT INTO `states` (`id`, `name`, `country_id`, `country_code`, `fips_code`, `
 (5218, 'Navassa Island', 234, 'UM', NULL, '76', 'island', 18.41006890, -75.01146120, '2023-03-25 20:08:55', '2023-03-25 20:08:55', 1, 'Q25359'),
 (5219, 'Palmyra Atoll', 234, 'UM', NULL, '95', 'island', 5.88850260, -162.07866560, '2023-03-25 20:08:55', '2023-03-25 20:08:55', 1, 'Q123076'),
 (5220, 'Wake Island', 234, 'UM', NULL, '79', 'island', 19.27961900, 166.64993480, '2023-03-25 20:08:55', '2023-03-25 20:08:55', 1, 'Q43296');
```

Comment:
This line should end with `,`

Reply:
Updated the mentioned line. Could you please approve the PR?